src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/TestCaseBuilder.java

```diff
@@ -46,6 +46,16 @@ static CaseDetails caseWithSearchCaseReference(Object searchCaseReferenceType, O
         return createCaseWith(b -> b.data(caseData));
     }
 
+    static CaseDetails caseWithAwaitingPaymentsAndClassification(
+        Object awaitingPaymentsProcessing,
+        Object classification
+    ) {
+        Map<String, Object> data = new HashMap<>();
+        data.put("awaitingPaymentDCNProcessing", awaitingPaymentsProcessing);
+        data.put("journeyClassification", classification);
```

Comment:
are there constants for those somewhere?

Reply:
yes, updated with constants